pymc/printing.py

```diff
@@ -215,9 +217,20 @@ def _expand(x):
     names = [x.name for x in parents]
 
     if "latex" in formatting:
-        return r"f(" + ",~".join([r"\text{" + _latex_escape(n) + "}" for n in names]) + ")"
+        return (
+            r"f("
+            + ",~".join([_latex_text_format(_latex_escape(n.strip("$"))) for n in names])
+            + ")"
+        )
+    else:
+        return r"f(" + ", ".join([n.strip("$") for n in names]) + ")"
+
+
+def _latex_text_format(text: str) -> str:
+    if "operatorname" in text:
```

Comment:
```suggestion
    if "\\operatorname{" in text:
```

Reply:
I pushed this change and fixed the failing test. One reference value was wrong after merging.
Now, I check why the coverage goes down.